source/_integrations/group.markdown

```diff
@@ -155,6 +155,14 @@ To add or remove entities from an existing group, click on `Group options`, all
 
 ![Group members](/images/integrations/group/Group_members.png)
 
+### Group attributes
+
+These are the attributes available for a group. Note that when the group's members are all `unavailable` the group itself becomes `unavailable` and all attributes are removed.
```

Comment:
The latter seems a bug? As in, why would we remove the `entity_id` attribute when the group becomes unavailable?

Reply:
I've removed the comments related to this behavior.